docs/changelog.rst

```diff
@@ -4,15 +4,17 @@ Changelog
 Unreleased
 ----------
 
-* Add the option to only show progress on the dashboard. (`#107`_)
-* Import escape directly from markupsafe, instead of from flask. (`#106`_)
+* Added support for the ``rich`` progress bar style (`#96`_)
```

Comment:
Does PR #108 need to be listed here as well? (The changelog wasn't updated in that PR)